changelog/2082.feature.rst

```diff
@@ -0,0 +1 @@
+Changed default TLS ciphers being used to ciphers configured by the system for OpenSSL 1.1.1+ and SecureTransport.
```

Comment:
In this case, starting with "Changed" makes the changelog harder to understand, I had to go back to the PR to understand what it did.

In particular, the current version does not make clear that we're not defining new defaults, we're just letting OpenSSL 1.1.1 and SecureTransport do their thing. Also, I guess Linux distributions can change the OpenSSL 1.1.1 ciphers, which is why we're using the vague "system" term?

```suggestion
Stopped configuring default ciphers for OpenSSL 1.1.1+ and SecureTransport as their own defaults are already secure.
```